source/extensions/stat_sinks/metrics_service/grpc_metrics_service_impl.h

```diff
@@ -73,18 +73,19 @@ class MetricsServiceSink : public Stats::Sink {
 public:
   // MetricsService::Sink
   MetricsServiceSink(const GrpcMetricsStreamerSharedPtr& grpc_metrics_streamer,
-                     TimeSource& time_system);
+                     TimeSource& time_system, const bool report_counters_as_deltas);
```

Comment:
What do you think about sending `MetricsServiceConfig` instead of just the `bool`?

Reply:
@ramaraochavali Given we only needed the one value out of the config I decided to pass it. Over time if more values are needed I could see us passing the config, as is done in other places in the codebase. That being said, I can pass it if you prefer.

Reply:
Do not have a strong opinion on this - I was just thinking by passing `MetricsServiceConfig` would future proof this if we add additional config which would any way have to be processed by `MetricsSink`. But NBD, either way - Just leave it for now as is